index.html

```diff
@@ -1,15 +1,158 @@
+				<h3 class="hero-date">2022.01.06(THU)~07(FRI)</h3>
+				<h5 class="hero-join">@ Central Conferance hall of Marid and more</h5>
+			</div>
+		</div>
+	</section>
+<!-- PROGRAMME SECTIONS-->
+	<section id="courses">
+		<div class="inner-course container">
+			<h2 class="course-menu">Main Programe</h2>
+			<hr/>
+			<ul class="ul-courses">
+				<li class="course-item">
+					<i class="fas fa-sitemap"></i>
+					<h4>Architecture</h4>
+					<p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Delectus ad placeat temporibus repudiandae.</p>
+				</li>
+				<li class="course-item">
+					<i class="fas fa-users"></i>
+					<h4>Usabilty</h4>
+					<p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Illo placeat quis asperiores sed.</p>
+				</li>
+				<li class="course-item">
+					<i class="fas fa-stethoscope"></i>
+					<h4>Testing</h4>
+					<p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Aut ipsum labore necessitatibus incidunt!</p>
+				</li>
+				<li class="course-item">
+					<i class="fas fa-user-lock"></i>
+					<h4>Security</h4>
+					<p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Nihil nostrum cumque incidunt debitis.</p>
+				</li>
+				<li class="course-item">
+					<i class="fas fa-step-forward"></i>
+					<h4>Procedures</h4>
+					<p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Itaque quidem harum repellendus beatae.</p>
+				</li>
+			</ul>
+			<h3 class="more-courses">Join WTSQ summit 2022</h3>
+			<div class="div-program">
+				<h3 class="hole-program">SEE THE HOLE PROGRAM</h3>
+			</div>
+		</div>
+	</section>
+<!-- FEATURED SPEAKERS SECTIONS-->
+	<section id="intructors">
+		<div class="inner-istructors container">
+			<h3 class="instructors-title">Fetaured Speakers</h3>
+			<hr/>
+			<ul id="cards-render" class="ul-instructors">
+				<!-- JAVASCRIPT RENDERING -->
+			</ul>
+			<button type="button" class="btn-more">More <i class="fas fa-chevron-down"></i></button>
+			<button type="button" class="btn-less">Show Less <i class="fas fa-chevron-up"></i></button>
+		</div>
+	</section>
+<!--
+	<section id="intructors">
+		<div class="inner-istructors container">
+			<h3 class="instructors-title">Fetaured Speakers</h3>
+			<hr/>
+			<ul class="ul-instructors">
+				<li>
+					<div class="profile-img show-more">
+						<img class="name-img" src="images/isntructor-01.jpg" alt="intructor 1">
+					</div>
+					<div id="profile-info" class="show-more">
+						<h3>Jonh Smith</h3>
+						<h4 >Lorem ipsum dolor sit amet.</h4>
+						<hr/>
+						<h5>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eum, quod.</h5>
+					</div>
+				</li>
+			</ul>
+			<button type="button" class="btn-more">More <i class="fas fa-chevron-down"></i></button>
+			<button type="button" class="btn-less">Show Less <i class="fas fa-chevron-up"></i></button>
+		</div>
+	</section>
```

Comment:
- [ ] Please correct me if I'm wrong, but this piece of commented code doesn't serve any purpose (unlike the ones you used to improve readability like `<!-- PARTNERS SECTIONS-->` which are fine), and seem to have been used as a placeholder as you wrote the JS, kindly remove this, as pointed by HTML and CSS best practices

![image](https://user-images.githubusercontent.com/73140461/147150757-379fac7c-6f3a-41e4-8e5d-e7de0f1b2dd3.png)

Reply:
Thank you very much for your kind suggestions 😃 👍 .